examples/connectivity_classes.py

```diff
@@ -0,0 +1,96 @@
+"""
+.. _ex-connectivity-measures:
+
+==============================
+Using the connectivity classes
+==============================
+
+Compute different connectivity measures and then demonstrate
+the utility of the class.
+
+Here we compute the Phase Lag Index (PLI) between all gradiometers and showcase
+how we might interface with the connectivity class.
+"""
+
+# Authors: Adam Li <adam2392@gmail.com>
+#
+# License: BSD (3-clause)
+
+import mne
+from mne_connectivity import spectral_connectivity
+from mne.datasets import sample
+
+# %%
+# Set parameters
+data_path = sample.data_path()
+raw_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw.fif'
+event_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw-eve.fif'
+
+# Setup for reading the raw data
+raw = mne.io.read_raw_fif(raw_fname)
+events = mne.read_events(event_fname)
+
+# Add a bad channel
+raw.info['bads'] += ['MEG 2443']
+
+# Pick MEG gradiometers
+picks = mne.pick_types(raw.info, meg='grad', eeg=False, stim=False, eog=True,
+                       exclude='bads')
+
+# Create epochs for the visual condition
+event_id, tmin, tmax = 3, -0.2, 1.5  # need a long enough epoch for 5 cycles
+epochs = mne.Epochs(raw, events, event_id, tmin, tmax, picks=picks,
+                    baseline=(None, 0), reject=dict(grad=4000e-13, eog=150e-6))
+
+# Compute connectivity for band containing the evoked response.
```

Comment:
```suggestion
# Compute connectivity for the band that contains the evoked response (4-9 Hz).
```

Generally, it would of course be nicer to use some genuine oscillatory activity. There might be alpha desynchronization going on here and I think we show gamma activity in one example of MNE-Python as well?